### PR TITLE
Move the module to v2

### PR DESCRIPTION
**Pull request by LucasRoesler**
**What**
- The breaking API change in the queue rquires a major version bump

- I have tested this in IdP with a `replace` directive. `go mod` works as expected and the code can compile, after adapting to the API changes.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,13 +11,11 @@ require (
 	github.com/getkin/kin-openapi v0.14.0
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
-	github.com/go-sql-driver/mysql v1.4.1 // indirect
+	github.com/go-sql-driver/mysql v1.5.0 // indirect
```
Comment (pragmader) [inline go.mod:14]: I know it has nothing to do with us but what in a world would require the mysql driver in our dependencies?

File: go.mod [hunk 2]
```diff
@@ -11,13 +11,11 @@ require (
...
 	github.com/golang/protobuf v1.3.2
-	github.com/google/uuid v1.1.1 // indirect
 	github.com/gorilla/websocket v1.4.0
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/lib/pq v1.1.1
-	github.com/mattn/go-sqlite3 v1.11.0 // indirect
+	github.com/mattn/go-sqlite3 v1.14.2 // indirect
```
Comment (pragmader) [inline go.mod:18]: And sqllite O_O

**Review by pragmader — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (LucasRoesler) [reply go.mod:18]: 100% agree i don't know why it is there

Comment (LucasRoesler) [reply go.mod:18]:
```sh
➜  go-base git:(feature-prepare-for-v2) go mod why github.com/mattn/go-sqlite3
# github.com/mattn/go-sqlite3
github.com/contiamo/go-base/v2/pkg/data/managers
github.com/Masterminds/squirrel
github.com/Masterminds/squirrel.test
github.com/mattn/go-sqlite3
```

Comment (LucasRoesler) [reply go.mod:18]: very weird that it doesn't shake out the tests

**Pull request merged**